### PR TITLE
407 org provider site

### LINKED ISSUE #407: MultiOrganization entity
Create a DB entity that relates multiple organizations

thought: should we just rename the existing "community" entity, as this is _not_ part of point in time launch and already has related permissions object etcc

Currently blocked by OEC's lack of definition.

### PR DESCRIPTION
**Pull request by jlhogan**
closes #407 

THe last commit is just a hand wave at what might be a good place to add convenience interpretation of the permissions from the server; whenever we get to actually implementing features relying on this stuff (deleting roster data with new upload, displaying multi-provider/multi-site rosters, creating new enrollments) I would expect the UserContext stuff to change again depending on what is actually needed and most useful!

### CODE REVIEW

**Review by mmazanec22 — APPROVED** [review]

File: src/controllers/users.ts [hunk 1]
```diff
@@ -1,12 +1,14 @@
-import { User, Site, Organization } from '../entity';
+import { User, Site, Provider, Organization } from '../entity';
 import { getManager } from 'typeorm';
 
 export const addDataToUser = async (user: User) => {
-  const [sites, organizations] = await Promise.all([
+  const [sites, providers, organizations] = await Promise.all([
     getManager().findByIds(Site, user.siteIds || []),
+    getManager().findByIds(Provider, user.providerIds || []),
```
Comment (kmagida110) [inline src/controllers/users.ts:7]: Random question, why are there no orgs here?

File: client/src/shared/models/db/Community.ts [hunk 2]
```diff
@@ -1,7 +0,0 @@
-import { Organization } from '.';
```
Comment (kmagida110) [inline client/src/shared/models/db/Community.ts:1]: Will we be adding a model for a geographic location once we figure out what that would be, ZIP/Town/County etc.

**Review by kmagida110 — APPROVED** [review]
From what I know this looks like the right structure, I have a few questions but nothing blocking.

File: src/controllers/users.ts (hunk 1 above)
Comment (jlhogan) [reply src/controllers/users.ts:7]: b/c i messed up :)

File: client/src/shared/models/db/Community.ts (hunk 2 above)
Comment (jlhogan) [reply client/src/shared/models/db/Community.ts:1]: Not sure, right now in the system we only are collecting regions for sites which enabled funding reporting. Ultimately if this is going to be the system of record for all thngs provider, then yes we will need to track addresses here as well. I think in a dream world, this is the system of record for enrollment info and there is a separate one for provider info, and this talks to that. But that's so many sprints into a future that i think the answer is yes we should probably capture address data for all entities ? @RyanHansz

**Pull request closed without merging**